consistency-checker/src/main/resources/migrationJob.json

```diff
@@ -133,12 +133,12 @@
                             },
                             "description": "Actual end date for this migration job"
                         },
-                        "completedFlag": {
-                            "type": "boolean",
+                        "jobStatus": {
+                            "enum":[ "NEW","READY","STARTING","RUNNING","RUNNING_ASYNC","FINISHING","COMPLETED_SUCCESS","COMPLETED_PARTIAL","COMPLETED_FAILED","ABORTING","ABORTED_DUPLICATE","ABORTED_TIMEOUT","ABORTED_AUTH","ABORTED_UNKNOWN","UNKNOWN" ],
```

Comment:
Enums are defined in entityInfo.  See [this](https://github.com/lightblue-platform/lightblue-core/blob/master/metadata/src/test/resources/metadata-json-schema-test-valid/schema-test-metadata-object-object.json#L10-L15) for defining an enum and [this](https://github.com/lightblue-platform/lightblue-core/blob/master/metadata/src/test/resources/metadata-json-schema-test-valid/schema-test-metadata-object-object.json#L39-L44) for using the enum.